contrib/test_jenkins.sh

```diff
@@ -941,6 +941,16 @@ test_memtrack() {
 	UCX_MEMTRACK_DEST=stdout ./test/gtest/gtest --gtest_filter=test_memtrack.sanity
 }
 
+test_memtrack_limit() {
+	../contrib/configure-devel --prefix=$ucx_inst
+	make_clean
+	$MAKEP
+
+	echo "==== Running memtrack limit test ===="
+	UCX_MEMTRACK_DEST=stdout UCX_HANDLE_ERRORS=none UCX_MEMTRACK_LIMIT=512MB ./test/apps/test_memtrack_limit | grep -q 'SUCCESS'
```

Comment:
`... test_memtrack_limit |& grep -C 100 'SUCCESS'`
- use |& to redirect all output streams
- print surrounding lines to show more info in test log
(same for 'reached' grep)